spec/osut_tests_spec.rb

```diff
@@ -374,6 +375,25 @@
     expect(model).to_not be_empty
     model = model.get
 
+    # -- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- cleanup   #
+    # This is the first test relying on the 'seb.osm' file, which ships with
+    # OpenStudio. The v1.11.5 model (2016) holds a number of artefacts that
+    # would be frowned upon today (e.g. plenum floors tagged as roofs/ceilings).
+    plenum = model.getSpaceByName("Level 0 Ceiling Plenum")
+    expect(plenum).to_not be_empty
+    plenum = plenum.get
+
+    plenum.surfaces.each do |s|
+      next if s.outsideBoundaryCondition.downcase == "outdoors"
+
+      expect(s.setSurfaceType("Floor")).to be true
+      expect(s.setVertices(s.vertices.reverse)).to be true
+    end
```

Comment:
The antiquated plenum floor vertex sequencing starts to be an issue with more recent _OpenStudio_ versions.